oaknational/Oak-Web-Application
# Feat/lesq 684/specialist programme page [LESQ-684]

## Description

Music year: 1979

- create specialistProgrammeListingView component
- add queries required for specialist programme listing page
- add specialist programme listing page
- update styling for page

## How to test

Unfortunately this is another one that isn't properly testable on preview environments - everything should be the same

1. Go to https://deploy-preview-2299--oak-web-application.netlify.thenational.academy
2. check the lesson journey
3. go to https://deploy-preview-2299--oak-web-application.netlify.thenational.academy/teachers/specialist/physical-development/programmes and see a 404 (internal server error)

## Screenshots

How it should now look:

![Screenshot 2024-03-06 at 15 09 27](https://github.com/oaknational/Oak-Web-Application/assets/45038878/6acfd7ad-3b25-4745-9c0e-b5542e2d1cc2)
  

![Screenshot 2024-03-06 at 15 09 50](https://github.com/oaknational/Oak-Web-Application/assets/45038878/0b7e8c4e-b2ac-449f-b9ee-7c7e1d512562)
  
  

![Screenshot 2024-03-06 at 15 17 06](https://github.com/oaknational/Oak-Web-Application/assets/45038878/dbe1c8c5-1a46-4de9-9522-c6958a1dbc37)
